src/main/java/blackjack/domain/gamer/Dealer.java

```diff
@@ -1,53 +1,62 @@
 package blackjack.domain.gamer;
 
-import blackjack.domain.card.Cards;
-import blackjack.domain.Outcome;
 import blackjack.domain.card.Card;
+import blackjack.domain.card.Cards;
+import blackjack.domain.card.Score;
+import java.util.Collections;
 import java.util.List;
 
-public class Dealer extends Participant {
-    private static final int MAX_AVAILABLE_TO_GET_CARD = 16;
+public class Dealer implements Participant {
+    private static final Score MINIMUM_SCORE_OF_NOT_TAKING_CARD = Score.of(17);
+    private final Name name;
+    private final Cards cards;
+
+    public Dealer() {
+        this(new Name("딜러"), new Cards(Collections.emptyList()));
+    }
+
+    public Dealer(Cards cards) {
+        this(new Name("딜러"), cards);
+    }
 
-    public Dealer(List<Card> cards){
-        super("딜러", cards);
+    public Dealer(Name name, Cards cards) {
+        this.name = name;
+        this.cards = cards;
     }
 
     @Override
-    public boolean isAvailableToTake() {
-        return super.sumCards() <= MAX_AVAILABLE_TO_GET_CARD;
+    public boolean isNotAbleToTake() {
+        return sumCards().isHigherThan(MINIMUM_SCORE_OF_NOT_TAKING_CARD);
```

Comment:
하나씩 선언하는 것이 가독성이 더 좋아 보입니다 :)
```suggestion
        Score score = sumCards();
        return score.isHigherThan(MINIMUM_SCORE_OF_NOT_TAKING_CARD);
```